mail-client/neomutt-test-files/neomutt-test-files-20200626.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+UPSTREAM_COMMIT="8629adab700a75c54e8e28bf05ad092503a98f75"
+SRC_URI="https://github.com/neomutt/${PN}/archive/${UPSTREAM_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+DESCRIPTION="Test files for the main NeoMutt source"
+HOMEPAGE="https://neomutt.org/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+BDEPEND=""
```

Comment:
These are all just waste of space in the ebuild and can be removed if you wish so.

Reply:
Done.